packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -70,41 +76,46 @@ The expandable list adds an expansion panel to every row to reveal more details
 Use an expandable data list when you have more information than will comfortably fit inside a row, or you want to provide a way for advanced users to access information that is not applicable to all users.
 
 ## Tables
-* **Standard table:** Combines the styling of a list view with behaviors that are commonly associated with a table (sometimes referred to as a hybrid list)
```

Comment:
How come you deleted the standard table and compact table info! I think it's useful to have it!
Would at a minimum have a similar description that you had in the data list section, so something simple like:
"PatterFly supports two main types of table"
- Compact  tables
- Standard tables"